.codeocean/settings.json

```diff
@@ -4,5 +4,10 @@
 	"role": {
 		"role_arn": "arn:aws:iam::467914378000:role/aind-codeocean-user"
 	},
+	"nextflow_error_strategy": {
```

Comment:
Is that low number of retries indeed what you want? (I haven't been aware that this is in settings.json, maybe it doesn't matter and the nextflow.config takes precedence? 🤷‍♂️)

Reply:
fixed